frontend/src/features/properties/components/forms/subforms/PagedBuildingForms.tsx

```diff
@@ -58,6 +73,22 @@ const PagedBuildingForms: React.FC<PagedBuildingFormsProps> = (props: PagedBuild
         name="buildings"
         render={arrayHelpers => (
           <div>
+            {showDelete && (
```

Comment:
you may not need this conditional render if you are using the display prop.

Reply:
So I removed this and it worked functionally fine, but I was getting a bunch of failures on the unit tests...